backend/app/well_known.py

```diff
@@ -0,0 +1,59 @@
+from typing import List
+
+from app.settings import conf
+from fastapi import APIRouter, Request, Response
+from pydantic import AnyUrl, BaseModel
+
+router = APIRouter()
+
```

Comment:
Maybe somewhere here in the top of the file explain that this is used to set up a party-configuration and the keys for it and a link to the [well-known-docs](https://well-known-docs.sandbox.ioxio-dataspace.com/party-configuration.html) and [guides](https://ioxio.com/guides/how-to-create-a-group#creating-and-hosting-party-configuration) so people can read more?